firebolt_provider/operators/firebolt.py

```diff
@@ -137,6 +143,8 @@ def __init__(
         self.firebolt_conn_id = firebolt_conn_id
         self.engine_name = engine_name
         self.database = None
+        self.query_timeout = None
+        self.fail_on_query_timeout = False
```

Comment:
Should this be `True` by default?

Reply:
I think it doesn't matter since we won't support engine start/stop timeout here so it will not be accounted for. I can set it to true

Reply:
Let's set it to true just to avoid confusion, users might see it and have questions why can't they fail if something goes wrong. If there's any timeout behind the scenes (e.g. network disconnect or something) we'll propagate it.